js/game.js

```diff
@@ -0,0 +1,36 @@
+var rats = document.querySelectorAll('.rat');
+
+/**
+ * When you click on a rat, it disappears.
+ *
+ * @param {Node} rat The rat that is clicked.
+ */
+function whack(rat) {
+  rat.classList.add('hidden');
+}
+
+rats.forEach(function(rat) {
+  rat.addEventListener('click', function() {
+    whack(rat);
+  });
+});
+
+/**
+ * Timer
+ *Timer which counts from 30 to 0 and then shows TIMES UP!
+ * @type {number}
+ */
+var count = 30;
```

Comment:
I would rename this to something more semantic - maybe timer instead of count? Makes it clearer what the variable is